src/logging.jl

```diff
@@ -1,6 +1,10 @@
 @static if VERSION < v"1.0-"
     import Base: info, warn
 end
+
+@static if VERSION >= v"0.7"
+    using Printf
```

Comment:
Using a package without installing it will not work.

Reply:
`Printf` is in the standard library.